mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -376,6 +377,14 @@ properties:
                         description: |
                           Number of accelerator cards exposed to the instance.
                         required: true
+          - !ruby/object:Api::Type::KeyValuePairs
```

Comment:
looks like it's having issues when building the provider. Could you address the issues when building?